src/repository/entity/OwnerEntity.java

```diff
@@ -0,0 +1,75 @@
+package repository.entity;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+public class OwnerEntity extends BaseEntity {
+
+    public OwnerEntity() {
+    }
+
+    public OwnerEntity(UUID ownerId, String name, LocalDate birthDate) {
```

Comment:
It is better to leave names of super class fields in constructor as they are
No need to switch from 'id' to 'ownerId'